<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LanguageChooser</class>
 <widget class="QDialog" name="LanguageChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Language</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="spanishLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image:url(:/languages/spanish-flag.png);</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QRadioButton" name="radioSpanish">
         <property name="styleSheet">
          <string notr="true">font: bold;font-size: 18px;</string>
         </property>
         <property name="text">
          <string>Espa√±ol</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="englishLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image:url(:/languages/english-flag.png);</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QRadioButton" name="radioEnglish">
         <property name="styleSheet">
          <string notr="true">font: bold;font-size: 18px;</string>
         </property>
         <property name="text">
          <string>English</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="germanLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image:url(:/languages/german-flag.png);</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QRadioButton" name="radioGerman">
         <property name="styleSheet">
          <string notr="true">font: bold;font-size: 18px;</string>
         </property>
         <property name="text">
          <string>Deutsch</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="styleSheet">
        <string notr="true">font: bold;font-size: 18px;</string>
       </property>
       <property name="text">
        <string>OK / Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
